lab8/duplicates.py

```diff
@@ -0,0 +1,46 @@
+import os, hashlib
+
+# function which change file values into 128 bit hash value
+# then for checking the duplicates
+def hashfile(path, size=4500):
+    with open(path, 'rb') as f:
+        hasher = hashlib.md5()
+        file = f.read(size)
+        while len(file) > 0:
+            # changing into 128 bit hash value
+            hasher.update(file)
```

Comment:
you have to accumulate hashes on just replace, you need to hash file-stream here